downstream/modules/devtools/proc-rhdh-set-up-controller-project.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="rhdh-set-up-controller-project_{context}"]
+= Setting up a controller project to run your playbook project
+
+.Procedure
+
+. The Ansible plug-ins provide a link to {PlatformNameShort}.
+. Log in to your {RHDH} UI.
+. Click the Ansible `A` icon in the {RHDH} navigation panel.
+. Click *Operate* to display a link to your {PlatformNameShort} instance.
++
+If {ControllerName} was not included in your plug-in installation, a link to the product feature page is displayed.
+. Click *Go to {PlatformNameShort}* to open your platform instance in a new browser tab.
++
+Alternatively, if your platform instance was not configured during the Ansible plug-in installation, navigate to your {ControllerName} instance in a browser and log in.
+. Log in to {ControllerName}.
+. Create a project in {PlatformNameShort} for the GitHub repository where you stored your playbook project.
+Refer to the
+link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_controller_user_guide/index#controller-projects[Projects]
+chapter of the _Automation controller user guide_.
+. Create a job template that uses a playbook from the project that you created.
+Refer to the
+link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_controller_user_guide/index#controller-create-job-template[Job Templates]
```

Comment:
```suggestion
link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/using_automation_execution/index#controller-job-templates[Job templates]
```

Reply:
@EMcWhinn Is this for 2.5?
The content needs to go in 2.4 also